pppd/peap.h

```diff
@@ -0,0 +1,76 @@
+/*
+ *      Copyright (c) 2011
+ *
+ * Authors:
+ *
+ *	Rustam Kovhaev <rkovhaev@gmail.com>
+ */
+
+#ifndef PPP_PEAP_H
+#define	PPP_PEAP_H
+
+#define	PEAP_PHASE_1			1
+#define	PEAP_PHASE_2			2
+
+#define	PEAP_HEADERLEN			6
+#define	PEAP_FRAGMENT_LENGTH_FIELD	4
+#define	PEAP_FLAGS_FIELD		1
+#define	PEAP_FLAGS_ACK			0
+
+#define PEAP_CAPABILITIES_TYPE		254
```

Comment:
Different indentation along the entire list of `#define`s

Reply:
Can be fixed.

Reply:
Looks like this file already does tab stop at 8 and is consistent. I had to set vim to ts=8 and the file displayed correctly. I could be wrong, but I am not seeing the problem here.

Reply:
This is what it looks like for me on GitHub:
![Screen Shot 2021-08-04 at 2 10 36 pm](https://user-images.githubusercontent.com/654521/128120935-b020cf1e-991f-485f-8bd2-c16757258708.png)

Reply:
Yes, the GitHub display seems wrong